rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/RestClient.java

```diff
@@ -67,15 +67,16 @@
     InputStream policies(@HeaderParam("sessionid") String sessionId);
 
     @POST
-    @Path("/rm/nodesource/create")
+    @Path("/rm/nodesource/create/recovery")
```

Comment:
by adding /recovery do we still have the /rm/nodesource/create rest endpoint working fine?

Reply:
Yes `rm/nodesource/create` endpoint still exists on the RM side but the RM portal does not use it anymore.